tla/consensus/abs.tla

```diff
@@ -4,22 +4,36 @@
 
 EXTENDS Sequences, SequencesExt, Naturals, FiniteSets, FiniteSetsExt
 
-CONSTANT Servers, Terms, MaxLogLength
+CONSTANT Servers
+ASSUME IsFiniteSet(Servers)
+
+\* Terms is a totally ordered set with a smallest element.  For simplicity, we use the naturals.
+CONSTANT Terms
+ASSUME \A a,b \in Terms : 
```

Comment:
Is it necessary to have all of this, can we not have something like:

```
ASSUME \A term \in Terms : term \in Nat
ASSUME \E minTerm \in Terms: \A term in Terms: minTerm <= term
```

Reply:
To simplify things, we could assume that Terms is equal to Nat (i.e., Terms \subseteq Nat). However, it is not necessary for Terms to be restricted to natural numbers; it can be any set that meets the specified properties.

Reply:
https://github.com/microsoft/CCF/pull/6475/commits/a69fa97a6fea111c2923255319e57a41f73cb14c